packages/conf-libmosquitto/conf-libmosquitto.1/opam

```diff
@@ -3,9 +3,13 @@ maintainer: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
 authors: [ "Markus W. Weissmann <markus.weissmann@in.tum.de>" ]
 homepage: "https://mosquitto.org/"
 license: [ "Eclipse Public License v1.0" "Eclipse Distribution License 1.0" ]
-build: [["ls" "/usr/include/mosquitto.h"]]
+build: [
+  ["ls" "/usr/include/mosquitto.h" {os != "darwin"}]
+  ["brew" "ls" "--versions" "mosquitto" {os = "darwin"}]
+]
 depexts: [
   [["debian"] ["libmosquitto-dev" ]]
   [["ubuntu"] ["libmosquitto-dev"]]
   [["alpine"] ["mosquitto-dev"]]
+  [["darwin"] ["mosquitto"]]
```

Comment:
This needs to be:

```
[["osx" "homebrew"] ["mosquitto"]]
```

in the depexts field

Reply:
Pull request updated.